networking_nsxv3/common/config.py

```diff
@@ -198,16 +164,11 @@
         help="NSXv3 Manager DFW connectivity strategy: {}"\
             .format(str(nsxv3_dfw_connectivity_strategy))
     ),
-    cfg.BoolOpt('nsxv3_groups_disconnect',
-        default=False,
-        help="Remvoe dynamic membership criteria from security groups."
-        # NSX-T objects will be created but not enforced on the segment ports
-    ),
     cfg.IntOpt(
-        'nsxv3_remove_orphan_items_count',
+        'nsxv3_cache_refresh_window',
         default=30,
-        help="Amount of items to be deleted if per sync loop."
-    ),
+        help="Amount of seconds transferred from cache history on refresh"
```

Comment:
Does "transferred" mean "removed" here? If not: transferred where?

Reply:
It actually transferred. When cache rebuilding is triggered there might be active jobs that will modify it, thus the transaction history has to be transferred on top of the regenerated cache.